packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,11 @@
   "pci_app_submit_select_image_preset_info": "Description",
   "pci_app_submit_select_image_advanced_tips": "Déployer votre propre application",
   "pci_app_submit_select_image_advanced": "Déployer votre propre application (image Docker)",
-  "pci_app_submit_select_image_tips": "Renseigner le chemin d'accès de votre image Docker. Cela peut être un registre public (GitHub, Gitlab, DockerHub, ...) ou privé",
+  "pci_app_submit_select_image_tips": "Renseigner le chemin d'accès de votre image Docker. Cela peut être un registre public (GitHub, Gitlab, DockerHub, ...) ou privé.",
+  "pci_app_submit_select_image_info_message_1": "Veillez à configurer votre registre si celui-ci est privé.",
+  "pci_app_submit_select_image_info_message_2": "Veuillez noter que l'image doit être construite avec une architecture AMD {{architectures}}. L'utilisation de docker-compose n'est pas possible.",
+  "pci_app_submit_select_image_info_message_3": "Veillez à donner les bons droits d'accès à l'utilisateur OVHcloud (user=42420:42420) afin que votre image se lance correctement:",
```

Comment:
```suggestion
  "pci_app_submit_select_image_info_message_3": "Veillez donner les bons droits d'accès à l'utilisateur OVHcloud (user=42420:42420) afin que votre image se lance correctement:",
```

Reply:
Discussed together, the wording is as expected

Reply:
as discuss together, you have right it's a good phrase.
No changes required